streaming/sender.c

```diff
@@ -480,6 +482,111 @@ static inline bool rrdpush_sender_validate_response(RRDHOST *host, struct sender
     return false;
 }
 
+unsigned char alpn_proto_list[] = {
+    18, 'n', 'e', 't', 'd', 'a', 't', 'a', '_', 's', 't', 'r', 'e', 'a', 'm', '/', '2', '.', '0',
+    8, 'h', 't', 't', 'p', '/', '1', '.', '1'
+};
+
+#define CONN_UPGRADE_VAL "upgrade"
+
+static int rrdpush_http_upgrade_prelude(RRDHOST *host, struct sender_state *s) {
+
+    char http[HTTP_HEADER_SIZE + 1];
+    int eol = snprintfz(http, HTTP_HEADER_SIZE,
```

Comment:
We have this warning:

```sh
streaming/sender.c: In function 'rrdpush_http_upgrade_prelude':
streaming/sender.c:495:9: warning: unused variable 'eol' [-Wunused-variable]
  495 |     int eol = snprintfz(http, HTTP_HEADER_SIZE,
```
during compilation time.

Reply:
done